jobclient/python/cook/scheduler/client/__init__.py

```diff
@@ -67,29 +68,24 @@ def __init__(self, host: str, port: int, *,
         :type host: str
         :param port: The port at which the Cook instance is listening.
         :type port: int
-        :param job_endpoint: The endpoint to be reached when scheduling new
-            jobs or getting the status of running jobs. Defaults to `/jobs`.
-        :type job_endpoint: str, optional
-        :param delete_endpoint: The endpoint to be reached when terminating
-            running jobs. Defaults to `/rawscheduler`
         """
-        self.__netloc = f'{host}:{port}'
-        self.__job_endpoint = job_endpoint
-        self.__delete_endpoint = delete_endpoint
+        self.__netloc = url
+        self.__job_endpoint = _DEFAULT_JOB_ENDPOINT
+        self.__delete_endpoint = _DEFAULT_DELETE_ENDPOINT
 
     def submit(self, *,
                command: str,
                cpus: float,
                mem: float,
                max_retries: int,
```

Comment:
Maybe we should make these have defaults too? 

The CLI defaults these so that you can so something as simple as:

```
cs submit env
```